configs/python/backend/backend.yaml

```diff
@@ -634,7 +634,7 @@ scanners:
           - '*'
       priority: 5
       options:
-        location: '/etc/yara/'
+        location: '/etc/strelka/yara/rules.yara'
```

Comment:
I'm good with this, although I'll give this a specific callout in the CHANGELOG.